lib/generators/sorcery/templates/initializer.rb

```diff
@@ -88,11 +88,18 @@
   #
   # config.ca_file =
 
+  # Linkedin requires r_emailaddress scope to fetch user's email address.
+  # You can skip it if email field is not mandatory in your database (by default it is).
```

Comment:
You can also skip including the email field if you use an intermediary signup form. (using `build_from` method)

The `r_emailaddress` scope is only necessary if you are using the `create_from` method directly.

Reply:
Adjusted 👍

Reply:
Thanks!